src/main/java/core/basesyntax/handler/impl/ReturnTransactionHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.handler.impl;
+
+import core.basesyntax.handler.TransactionHandler;
+import java.util.Map;
+
+public class ReturnTransactionHandler implements TransactionHandler {
+    @Override
+    public void handleTransaction(Map<String, Integer> mapFruitQuantity, String key, int value) {
+        mapFruitQuantity.put(key, mapFruitQuantity.getOrDefault(key, 0) + value);
```

Comment:
It's good that you're using `getOrDefault` to handle cases where the key might not be present in the map. However, please also consider the encapsulation principle. Map<String, Integer> should be encapsulated within the service that interacts with it, and not passed around and modified by various handlers.

Reply:
you can also use hashmap's merge with Integer::sum